src/Compilers/Core/Portable/Emit/EditAndContinue/DeltaMetadataWriter.cs

```diff
@@ -645,6 +677,60 @@ protected override void CreateIndicesForNonTypeMembers(ITypeDefinition typeDef)
             implementingMethods.Free();
         }
 
+        private SymbolChange FixSymbolChangeForReAddedMembers(IDefinition item, SymbolChange change)
```

Comment:
>FixSymbolChangeForReAddedMembers [](http://example.com/codeflow?start=29&length=32)

Can this be moved to `SymbolChanges` and named `GetDeletedSymbolChange`?

Reply:
Yeah, could do that, would just need the `isDeleted` local function to be a Func parameter.